arm/ui/templates/index.html

```diff
@@ -3,78 +3,92 @@
+                                                        <div class="col-lg-4">
+                                                                <div class="card-body px-1 py-1">
+                                                                        {% if not job.title_manual  %}
+                                                                        <b>{{  job["title"]  }}</b><br>
+                                                                        {% else %}
+                                                                        <del><b>{{  job["title_auto"]  }}</b><br></del>
+                                                                        <b>{{  job["title"]  }}</b><br>
+                                                                        {% endif %}
+                                                                        <b>Type: </b> {{ job.video_type }} <br>
+                                                                        {% if not job.title_manual %}
+                                                                        <b>Year: </b>({{ job["year"] }})<br>
+                                                                        {% else %}
+                                                                        <del><b>Year:
+                                                                                </b>{{ job["year_auto"] }}<br></del>
+                                                                        <b>Year: </b> {{  job["year"]  }}<br>
+                                                                        {% endif %}
+                                                                        <b>Device: </b>{{ job.devpath }}<br>
+                                                                        <b>Status: </b>{{ job.status }} <br>
+                                                                        <a href="titlesearch?job_id={{ job["job_id"] }}"
+                                                                                class="btn btn-primary">Title Search</a>
+                                                                        </li>
+                                                                </div>
+                                                        </div>
+                                                        <div class="col-lg-4">
+                                                                <div class="card-body px-1 py-1">
+                                                                        <b>Rip Method: </b>{{ job.config.RIPMETHOD }}
+                                                                        <br>
+                                                                        <b>Main Feature:
+                                                                        </b>{{ job.config.MAINFEATURE }} <br>
+                                                                        <b>Min Length:
```

Comment:
Shoot, I didn't realize I had Prettier running and reformatted this whole file! :(
What I noticed in the 2.2 branch is the label here says Max length then it displays the min length, and vice versa.  Just made that little fix here.